cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/CloudStack.java

```diff
@@ -45,18 +45,27 @@ public class CloudStack {
 
     private final String coreUserData;
 
+    private final boolean multiAz;
```

Comment:
at this point it would be better to already populate the instance.az information instead multiaz flag